recipes/fslpy/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "fslpy" %}
+{% set version = "1.10.1" %}
+{% set sha256 = "d8af3c12a4cac263ad5cb75c84004606faaa99ec5d3c6bfd2f9d3e337726c785" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - immv          = fsl.scripts.immv:main
+    - imcp          = fsl.scripts.imcp:main
+    - imglob        = fsl.scripts.imglob:main
+    - atlasq        = fsl.scripts.atlasq:main
+    - atlasquery    = fsl.scripts.atlasq:atlasquery_emulation
+    - extract_noise = fsl.scripts.extract_noise:main
+
+requirements:
+  build:
+    - python
+    - pip
+    - sphinx 1.6.*
+    - sphinx_rtd_theme 0.*
+    - mock 2.*
+    - coverage 4.*
+    - pytest 3.*
```

Comment:
do you need pytest just to test your project or you really need at build time?

Reply:
and also mock, pytest-coverage, coverage

Reply:
No, I copied as-is from the setup.py (where I've bundled build+test deps together). Will change.

Reply:
Ah ok, I just saw your ``setup.py`` that is using the ``requirements-dev.txt`` to the setup process. Would be good to change that, but as your ``setup.py`` requires that it has to be here indeed.  :)